cmd/osm-controller/osm-controller.go

```diff
@@ -197,10 +197,12 @@ func main() {
 		events.GenericEventRecorder().FatalEvent(err, events.InitializationError, "Error creating informer collection")
 	}
 
-	k8sClient := k8s.NewClient(osmNamespace, osmMeshConfigName, informerCollection, policyClient, msgBroker)
+	k8sClient := k8s.NewClient(osmNamespace, osmMeshConfigName, informerCollection, policyClient, configClient, msgBroker)
```

Comment:
I think for the time being, we're looking to leverage the informer cache and reduce dependencies on raw clients. Originally, I was worried about the informer cache being stale, but I realized that all of our k8s query methods leveraged the informer cache anyway. Does that make sense?

Reply:
I updated `GetMeshRootCertificate` to use the informer cache. For `UpdateMeshRootCertificate` and `UpdateMeshRootCertificateStatus` I am still utilizing the raw config client.